setup.cfg

```diff
@@ -10,7 +10,7 @@ max-line-length = 120
 
 [pycodestyle]
 exclude = .git,.tox,migrations
-max-line-length = 88
+max-line-length = 120
```

Comment:
Why?

Reply:
Because it is the style standard we have adopted for all Python code at edX: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#syntax-and-organization

Reply:
https://github.com/edx/edx-documentation/pull/1848